components/admin_console/admin_definition.jsx

```diff
@@ -224,7 +226,9 @@ const AdminDefinition = {
         sectionTitleDefault: 'About',
         isHidden: it.any(
             it.configIsTrue('ExperimentalSettings', 'RestrictSystemAdmin'),
-            it.not(it.userHasReadPermissionOnResource('about')),
+            it.all(
+                it.not(it.userHasReadPermissionOnResource(RESOURCE_KEYS.ABOUT.EDITION_AND_LICENSE)),
```

Comment:
Are these `About` changes related to this PR?

Reply:
Sorry that should be gone now